setup.py

```diff
@@ -20,7 +20,10 @@
 try:
     pkg_resources.require(['pip>=18', 'setuptools>=45, <66'])
 except (pkg_resources.VersionConflict, pkg_resources.DistributionNotFound):
-    print('Try update pip/setuptools versions, for example, python3 -m pip install --upgrade pip setuptools==65.7')
+    print(
+        '\033[93mTry update pip/setuptools versions, for example, '
```

Comment:
Shall we make \033[93m things standalone variables? It's hard to understand.

Reply:
it's only used once here and is a standard asci code